plugins/modules/inventory_intent.py

```diff
@@ -2874,8 +2953,8 @@ def verify_diff_deleted(self, config):
 
             if udf_id is None:
                 self.status = "success"
-                msg = "Global UDF - {0} deleted from Cisco DNA Center and verified successfully.".format(field_name)
-                self.log(msg)
+                msg = "Global UDF '{0}' deleted from Cisco Catalyst Center and verified successfully.".format(field_name)
```

Comment:
msg = "Global UDF named '{0}' has been successfully deleted from Cisco Catalyst Center and the deletion has been verified.".format(field_name)

Reply:
added.